src/EditorFeatures/Core/ExternalAccess/VSTypeScript/Api/VSTypeScriptInlineRenameReplacementWrapper.cs

```diff
@@ -11,13 +11,13 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api
 {
     internal readonly struct VSTypeScriptInlineRenameReplacementWrapper
     {
-        private readonly InlineRenameReplacement _underlyingObject;
+        internal readonly InlineRenameReplacement UnderlyingObject;
```

Comment:
Why internal rather than private? I would have assumed we want this private so TypeScript can try to use it...

Reply:
`VSTypeScriptInlineRenameReplacementInfo` needs to unwrap it. TS can't use it since the type of the field is internal and not in ExternalAccess.